contrib/storage-hive/core/pom.xml

```diff
@@ -58,6 +58,10 @@
           <groupId>commons-codec</groupId>
           <artifactId>commons-codec</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
Is the exclusion necessary due to a version conflict or the dependency is not required?

Reply:
For a transitive dependency will it be better to use `DependencyManagement`? Please add comments with details.